scan/src/rpc/AccountHook.re

```diff
@@ -0,0 +1,44 @@
+let decodeBalances = json =>
+  JsonUtils.Decode.(json |> field("result", list(json => json |> field("amount", uamount))));
+let decodeStakes = json =>
+  JsonUtils.Decode.(
+    json |> at(["result", "total"], list(json => json |> field("amount", uamount)))
+  );
+
+let getBalanceStake = address => {
+  let addressStr = address |> Address.toBech32;
+  let json = AxiosHooks.use({j|distribution/delegators/$addressStr/rewards|j});
+
+  let balances = json |> Belt.Option.map(_, decodeStakes);
+  switch (balances) {
+  | None => None
+  | Some(x) => Some(x |> Belt_List.reduce(_, 0.0, (x, y) => x +. y))
+  };
+};
+
```

Comment:
Add getReward from this [endpoint](http://d3n.bandprotocol.com/distribution/delegators/band1p40yh3zkmhcv0ecqp3mcazy83sa57rgjp07dun/rewards)